### ISSUE TITLE
Our location option not use with global configuration

### ISSUE THREAD

**Issue by anartzdev** [mugan86/ng-leaflet#22]
This option not available with global config when set config in principal module that load configuration
```
const config: IConfigMap = {
  ourLocation: {
    active: true
  }
};

@NgModule({
  imports: [BrowserModule, NgLeafletModule.forRoot(config), AppRoutingModule],
  declarations: [
    AppComponent,
    GlobalConfigAllComponent,
    GlobalOverWriteComponent,
  ],
  bootstrap: [AppComponent],
})
export class AppModule {}
```
Now, we add new component that implement specify configs to overwrite global. In this case, we want use "ourLocation" control option and not add in config. Only change defaultLayer.
```
import { Component } from '@angular/core';
import { IConfigMap, tileLayers } from '@mugan86/ng-leaflet';

@Component({
  selector: 'app-external-layer-default',
  templateUrl: './map.component.html',
  styleUrls: ['./map.component.css'],
})
export class GlobalOverWriteComponent {
  config: IConfigMap = {
    defaultLayer: {
      map: tileLayers.baseLayers.openTopoMap.map,
      atribution: tileLayers.baseLayers.openTopoMap.atribution,
    },
  };
}
```
In this case, not appear "ourLocation" control option and must be appear
https://angular-leaflet-map-ce-angular-ej-2.stackblitz.io/global-overwrite

**Comment by anartzdev** [comment]
We must implement in **https://github.com/mugan86/ng-leaflet/blob/develop/projects/mugan86/ng-leaflet/src/lib/components/map/map.component.ts**

```
if (this.config!.ourLocation!! === undefined && this.defaultConfig.get().config.ourLocation) {
      this.config!.ourLocation= this.defaultConfig.get().config.ourLocation;
}
```

**Comment by mugan86** [comment]
Fix the problem. Update this fix in 0.4.0 version!
